src/components/TopicsCard.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { Link } from "react-router-dom";
+import ListGroup from "react-bootstrap/ListGroup";
+import Card from "react-bootstrap/Card";
+import { getDay, getHour } from "../Utils/utils";
+
+const TopicsCard = () => {
+	const [articles, setArticles] = useState({});
+	const slug = useParams();
+	const [loading, setLoading] = useState(true);
+
+	useEffect(() => {
+		fetch(`https://caragea-nc-news-backend.herokuapp.com/api/articles`)
+			.then((response) => response.json())
+			.then(({ articles }) => {
+				setArticles(articles);
+				setLoading(false);
+			});
+	}, []);
```

Comment:
Per our previous discussion I'd consider a parameterised query here so that you only get back the specified topics eg 'https://caragea-nc-news-backend.herokuapp.com/api/articles/topic={topic}' so that it's all being handled by the server. Apologies if I didn't make that clear when we discussed it. 

I'd also consider separating these requests into an api file so that all requests are kept together and imported when needed.